units/getty@.service.in

```diff
@@ -55,6 +55,8 @@ IgnoreSIGPIPE=no
 SendSIGHUP=yes
 ImportCredential=agetty.*
 ImportCredential=login.*
+ImportCredential=tty.virtual.%I.agetty.*:agetty.
+ImportCredential=tty.virtual.%I.login.*:login.
```

Comment:
Should global settings take precedence over instance-specific ones? (Question, I have no preference here)

Reply:
I think having per instance take precedence over global makes more sense. You could use the global ones to set a default and use the per instance one to override the default. Though in practice for this I think you'll always want to use the per instance ones and never the global ones.

Reply:
Then the order should be changed? Because `load_credential_glob` skips on EEXIST, rather than overriding existing ones

Reply:
Where would the `EEXIST` come from though? `write_credential()` does an atomic replace with `renameat()`, that will override and not give `EEXIST` right?